divisible.rb

```diff
@@ -0,0 +1,11 @@
+def array_sort
+    my_range = Array((1..100))
```

Comment:
Your program does everything the assignment asks for, so you're all good on that front. I have just a few nits and clarifications to make.

```suggestion
    my_range = (1..100)
```

You don't need to load all the numbers from 1 to 100 into memory first, and technically speaking, wrapping `Array( )` around it turns `my_range` into an Array object, not a Range object. 😉 

```
> puts (1..100).class
Range
> puts Array((1..100)).class
Array
```